README.md

```diff
@@ -109,7 +109,10 @@ There are two ways to control the behave test engine's selection of test cases t
 
 > Note that the `<tag>` arguments passed in on the command line **cannot** have a space, even if you double-quote the tag or escape the space. This is because the args are going through multiple layers shells (the script, calling docker, calling a script in the docker instance that in turn calls behave...). In all that argument passing, the wrappers around the args get lost. That should be OK in most cases, but if it is a problem, we have the `-i` option as follows...
 
-To enable full control over behave's behaviour (if you will...), the `-i <ini file>` option can be used to pass a behave "ini" format file into the test harness container. The ini file enables full control over the behave engine, add handles the shortcoming of not being able to pass tags arguments with spaces in them. See the behave configuration file options [here](https://behave.readthedocs.io/en/stable/behave.html#configuration-files). Note that the file name can be whatever you want. When it lands in the test harness container, it will be called `behave.ini`.
+To enable full control over behave's behaviour (if you will...), the `-i <ini file>` option can be used to pass a behave "ini" format file into the test harness container. The ini file enables full control over the behave engine, add handles the shortcoming of not being able to pass tags arguments with spaces in them. See the behave configuration file options [here](https://behave.readthedocs.io/en/stable/behave.html#configuration-files). Note that the file name can be whatever you want. When it lands in the test harness container, it will be called `behave.ini`. The default ini file is located in `aries-agent-test-harness/aries-test-harness/behave.ini`. To run the tests with the default behave.ini,
+ ```
+ ./manage run -d acapy -t @AcceptanceTest -t ~@wip -i aries-test-harness/behave.ini
```

Comment:
To run with the default behave.ini file, don't you just not include the `-i <file>` command?  I think that should be clarified in this.

Reply:
No, the default ini is currently not being picked up when running behave through the manage script. Even when the ini file is in the root aries-agent-test-harness folder where the manage script is, behave still doesn't seem to see it. Quickest fix for now was to just document to use the -i option to point to the behave.ini file, hence the change to the docs above. 

The best place for the behave.ini file is where it is, in the aries-agent-test-harness/aries-test-harness folder. This is because if you run behave without manage it works with the default ini and finds the features to execute in the immediate features folder. 

I didn't spend a ton of time on it, just wanted a quick doc fix and move on.